lib/modules/SmartVariable.coffee

```diff
@@ -8,7 +8,7 @@
     # -------------------------------------
     #  Variable Types
     # -------------------------------------
-        VARIABLE_PATTERN = '\\{{ VARIABLE }}[\\s]*\\:[\\s]*([^\\;\\n]+)[\\;|\\n]'
+        VARIABLE_PATTERN = '\\{{ VARIABLE }}[\\s]*[:|=][\\s]*([^\\;\\n]+)[\\;|\\n]'
```

Comment:
There `|` has no use inside a bracket (it doesn't mean `OR`, as inside a capturing group): In this case, you'd also, unintentionally, match
```css
$variable|value
```